src/main/webapp/app/entities/oauth-client/oauth-client-dialog.component.html

```diff
@@ -0,0 +1,75 @@
+            <small id="scopeHelp" class="form-text text-muted">Comma seperated list of scopes. Scopes should have the following structure: ENTITY.OPERATION. See the <a href="https://github.com/RADAR-CNS/ManagementPortal/tree/master/radar-auth">radar-auth library documentation</a> for more information.</small>
+        </div>
+        <div class="form-group">
+            <label class="form-control-label" for="" jhiTranslate="managementPortalApp.oauthClient.resourceIds">Resources</label>
+            <input type="text" class="form-control" id="resourceIds" name="resourceIds" [(ngModel)]="resourcesList" [disabled]="protectedClient" required />
+            <small id="scopeHelp" class="form-text text-muted">Comma seperated list of allowed resource IDs.</small>
+        </div>
+        <div class="form-group">
+            <label class="form-control-label" for="" jhiTranslate="managementPortalApp.oauthClient.authorizedGrantTypes">Grant Types</label>
+            <div class="form-check" *ngFor="let grant of grantTypes">
+                <label class="form-check-label">
+                    <input type="checkbox" class="form-check-input" name="grantTypes" [checked]="grant.authorized" (change)="grant.authorized = !grant.authorized" [disabled]="protectedClient" /> {{grant.name}}
+                </label>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="form-control-label" for="" jhiTranslate="managementPortalApp.oauthClient.autoApprove">Auto Approve</label>
+            <input type="text" class="form-control" id="autoApprove" name="autoApprove" [(ngModel)]="autoApproveScopeList" [disabled]="protectedClient" />
+            <small id="autoApproveHelp" class="form-text text-muted">Comma seperated list of scopes that do not require explicit user approval in the authorization code grant.</small>
+        </div>
+        <div class="form-group">
+            <label class="form-control-label" for="" jhiTranslate="managementPortalApp.oauthClient.accessTokenValidity">Access Token Validity</label>
+            <input type="text" class="form-control" id="accessTokenValidity" name="accessTokenValidity" [(ngModel)]="client.accessTokenValidity" [disabled]="protectedClient" required pattern="\d+" />
+        </div>
+        <div class="form-group">
+            <label class="form-control-label" for="" jhiTranslate="managementPortalApp.oauthClient.refreshTokenValidity">Refresh Token Validity</label>
+            <input type="text" class="form-control" id="refreshTokenValidity" name="refreshTokenValidity" [(ngModel)]="client.refreshTokenValidity" [disabled]="protectedClient" required pattern="\d+" />
+        </div>
+        <div class="form-group">
+            <label class="form-control-label" for="" jhiTranslate="managementPortalApp.oauthClient.additionalInformation">Additional Information</label>
+            <div class="form-check">
+                    <label class="form-check-label">
```

Comment:
I like the idea of additional information. Can we also add `protected-clients` here. so that we can have secured clients too.